app/views/abuse_reports/new.html.erb

```diff
@@ -40,7 +41,8 @@
         <%= f.label :language, t(".form.language.label") %>
       </dt>
```

Comment:
It looks like there's a lot of selects where we take an array of languages or locales and turn them into options like:
```
@abuse_languages.map { |language| [language.name, language.name, { lang: language.short }] }
@news_languages.map { |language| [language.name, language.id, { lang: language.short }] }
@news_languages.map { |language| [language.name, language.short, { lang: language.short }] }
@languages.collect { |language| [language.name, language.id, { lang: language.short }] }

Locale.default_order.map { |locale| [locale.name, locale.iso, { lang: locale.language.short }] }
@available_locales.collect { |locale| [locale.name, locale.id, { lang: locale.language.short }] }
```

Is there any way we could use a helper or two to make that a little less repetitive, e.g., a method that takes an array of languages (or locales) and lets you specify whether to use `id`, `short`, or `name` as the option's value?